website/docs/verifydeprecatedapi.md

```diff
@@ -0,0 +1,759 @@
+---
+id: verifydeprecatedapi
+title: Verify deprecated APIs
+---
+
+# Verify deprecated APIs
+
+## Description
+Verifies deprecated Kubernetes APIs to ensure all the API versions are up to date.
+
```

Comment:
can we add note saying this won't be applicable for audit and state the reason

Reply:
done!